src/carousel/carousel.scss

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-carousel {
+  @include iui-reset;
+  display: flex;
+
+  .bnt-hc-scroll-container {
```

Comment:
What is this class name? :D

Reply:
It's the classnames Alexandr was using.  I didn't bother changing the class names until we figure out what we want to call this component.  Any suggestions?  Carousel seems like the whole component (dots, arrow buttons, sliding part).  Maybe we call it `.iui-carousel-navigation`?  I could maybe use the `<nav>` then?

Reply:
I would have expected to have full carousel component with sliding animation of the elements. I do not really get only the dots part. Especially, when we have that document already in ux standards (welcome page).

Reply:
the dots are used in both loading page and welcome page, which look different.

| loading | welcome |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/9084735/141321139-6cfbccb2-023f-474d-a8ea-c1dc927d953a.png" width="600px" /> | <img src="https://user-images.githubusercontent.com/9084735/141321108-6097ffd8-b6ad-4f6a-b85f-dbc7b1d54e4e.png" width="500px" /> |

I think it should be extracted out into something reusable, since it could be used in other places and it has so much logic in it anyway (both CSS and JS).

Reply:
I find both loading and welcome page carousels same component.

Reply:
I wanted to say that loading, welcome and Alexandr's gallery components are not the same. But thought that using composition in React for bottom content, I think it is possible to achieve all of them.